sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Azure.Monitor.OpenTelemetry.AspNetCore.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What has changed here?

Reply:
This usually means the file encoding changed.
When Powershell scripts edit our csproj files, it seems to apply one encoding. And then when we edit files in Visual Studio it sometimes applies a different encoding.